src/django/api/processing.py

```diff
@@ -33,7 +36,53 @@ def _report_error_to_rollbar(file, request):
                 'file_name': file.name})
 
 
-def get_excel_sheet(file, request):
+def get_xlsx_sheet(file, request):
+    import defusedxml
+    from defusedxml.common import EntitiesForbidden
+
+    defusedxml.defuse_stdlib()
+
+    try:
+        wb = load_workbook(filename=file)
+        ws = wb[wb.sheetnames[0]]
+
+        return ws
+
+    except EntitiesForbidden:
+        _report_error_to_rollbar(file, request)
+        raise ValidationError('This file may be damaged and '
+                              'cannot be processed safely')
+
+
+def parse_xlsx(file, request):
+    try:
+        ws = get_xlsx_sheet(file, request)
+
+        percent_col = []
+
+        for cell in ws[2]:
+            if '%' in cell.number_format:
+                percent_col.append(get_column_letter(cell.column))
+
+        if percent_col:
+            for col in percent_col:
+                for cell in ws[col]:
+                    if cell.row != 1:
+                        cell.value = f"{cell.value * 100}%"
+
+        header = ','.join([cell.value for cell in ws[1]])
+
+        rows = ['"{}"'.format(
+            '","'.join([str(cell.value) for cell in ws[idx]]))
+                for idx in range(2, ws.max_row)]
+
+        return header, rows
+    except Exception:
+        _report_error_to_rollbar(file, request)
+        raise ValidationError('Error parsing Excel (.xlsx) file')
+
+
+def get_xls_sheet(file, request): 
```

Comment:
The OAR team confirmed that we can drop support for `.xls` files.  I like the new `parse_xlsx` name so we don't have to revert back to the old `parse_excel`, just remove the methods that deal with xls. 

In addition to removing the handling here we will also want to remove xls from the extensions list in `ContributeForm`

https://github.com/open-apparel-registry/open-apparel-registry/blob/77717f0046a5b0c6b4d83966df2bfa3d4e2a3bf4/src/app/src/components/ContributeForm.jsx#L187